### DISCUSSION TITLE
Change callout color, or avoid overriding color of custom callouts

### DISCUSSION THREAD

**Discussion by matar3** [RyotaUshio/obsidian-pdf-plus#202] [Q&A] [answered]
Hi there,
I'm using the command "Quote in callout", and i'm trying to change the default color of the callouts created in Obsidian. I have a custom css file in obsidian snippet folder that lets me create a custom type of callout, that i called "blanko". However, if i write this type of callout in pdf++ settings, it still colors the callout yellow, overriding the color i'd set for this type of callout (i tried to disable option "use pdf++ callouts" but nothing changed, even though this thing should let us use our css. Even changing the default highlight color in pdf++ the color of callouts is still yellow) . Can't we modify the default settings of the callouts? Moreover, can we make callout collapsible by default?
![image](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/31188460/9c4c20ce-7bc6-428c-9c90-eafffdb0ffc0)

**Comment by RyotaUshio** [comment, marked as answer]
I guess you haven't read this page of the docs: https://ryotaushio.github.io/obsidian-pdf-plus/link-copy-formats.html

If you have your custom css for custom callout type, then just use it in your copy format and disable "Use PDF++ callout".

>  Moreover, can we make callout collapsible by default?

You can do so by editing the copy format "Quote in callout". See the linked page of the docs for the details.

**Reply by matar3** [reply]
I did have read that page of the documentation, but could not understand where to apply it, i.e. where to write the customized settings, like the default callout type to use. Then i found it in the settings, and it is a section called "Custom link copy formats". It is not so straight the passage between the documentation and the settings of the plugin: the settings are themselves clear, so best thing would be just to ease navigation through settings. Let me give you a suggestion: the icons you set in the upper bar of the settings are useful, but not enough: if the only way to see the settings is inside the Obsidian window, then it is difficult to find something, and above all it is impossible to make a full-text search in the page of the settings. I think you should write a link to html webpage with the content of the settings, so that one can read them on the browser instead than in a small window inside obsidian. In the beginning i thought that the documentation html was quite this thing, but then i understood it is instead an extra thing added to the settings, whereas the only way to read the settings is inside obsidian.

**Reply by RyotaUshio** [reply]
Reasonable suggestion, but it will make the maintenance a little harder because it requires a bit of work to keep the html page always up-to-date to the actual settings. (also please understand the current incomplete status of the docs is simply because of the lack of my time - writing a documentation website is such a demanding work)

>  it is difficult to find something, and above all it is impossible to make a full-text search in the page of the settings.

I'd suggest you try the Settings Search plugin.

(Also a reminder: this is an open source project maintained by an unpaid individual, and you always have the option to contribute if you are willing to. This applies not only to the code but also the docs.)

**Reply by matar3** [reply]
You could specify that the webpage is not regularly updated, so it is better to look at the settings in obsidian. However, i believe it would still be of great help to have a webpage with the settings (or even just a static pdf file), even if not updated, because it is sthg very different from the documentation.
P.S. I knew you are the developer of the project but i didn't think that you did everything by yourself and still keep updating the project and the docs by yourself. First of all, Congratulations ! Then, I think you could add some premium (paid) features if u need more support for your project.
